<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TerminalView</class>
 <widget class="QWidget" name="TerminalView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>427</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="TextEdit" name="textEditRx" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_19">
         <item>
          <widget class="QWidget" name="widget_5" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_16">
            <property name="spacing">
             <number>3</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QGroupBox" name="groupBox_9">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string>Rx Settings</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_17">
               <property name="spacing">
                <number>2</number>
               </property>
               <property name="leftMargin">
                <number>4</number>
               </property>
               <property name="rightMargin">
                <number>4</number>
               </property>
               <property name="bottomMargin">
                <number>4</number>
               </property>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_20">
                 <item>
                  <widget class="QRadioButton" name="portReadAscii">
                   <property name="text">
                    <string notr="true">ASCII</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="portReadHex">
                   <property name="text">
                    <string notr="true">Hex</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QCheckBox" name="wrapLineBox">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string>Wrap Line</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_10">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string>Tx Settings</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_18">
               <property name="spacing">
                <number>2</number>
               </property>
               <property name="leftMargin">
                <number>4</number>
               </property>
               <property name="rightMargin">
                <number>4</number>
               </property>
               <property name="bottomMargin">
                <number>4</number>
               </property>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_21">
                 <item>
                  <widget class="QRadioButton" name="portWriteAscii">
                   <property name="text">
                    <string notr="true">ASCII</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="portWriteHex">
                   <property name="text">
                    <string notr="true">Hex</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_22">
                 <property name="spacing">
                  <number>2</number>
                 </property>
                 <item>
                  <widget class="QCheckBox" name="resendBox">
                   <property name="text">
                    <string>Resend</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="resendIntervalBox">
                   <property name="minimum">
                    <number>10</number>
                   </property>
                   <property name="maximum">
                    <number>120000</number>
                   </property>
                   <property name="value">
                    <number>1000</number>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>ms</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_7" columnstretch="2,1,1">
           <property name="verticalSpacing">
            <number>3</number>
           </property>
           <item row="1" column="0">
            <widget class="QComboBox" name="historyBox"/>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="sendButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Send</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="3">
            <widget class="TextEdit" name="textEditTx" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="focusPolicy">
              <enum>Qt::StrongFocus</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QRadioButton" name="rCL">
             <property name="text">
              <string>加回车换行</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="3" column="1">
        <widget class="QPushButton" name="log2Button">
         <property name="text">
          <string>Log2</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="tstButton">
         <property name="text">
          <string>切换tst</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="dialButton">
         <property name="text">
          <string>呼叫112</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="urlButton">
         <property name="text">
          <string>设置url</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QPushButton" name="gpsRun">
         <property name="text">
          <string>GPS运行间隔</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QPushButton" name="gpsSignal">
         <property name="text">
          <string>GPS信号</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="paramButton">
         <property name="text">
          <string>参数</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="iccidButton">
         <property name="text">
          <string>ICCID</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="closelog">
         <property name="text">
          <string>关log</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="imeiButton">
         <property name="text">
          <string>读IMEI</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="pwoffButton">
         <property name="text">
          <string>关机</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="writeimei">
         <property name="text">
          <string>写IMEI</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="log1Button">
         <property name="text">
          <string>Log1</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="hungButton">
         <property name="text">
          <string>挂断电话</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="rebootButton">
         <property name="text">
          <string>重启</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QPushButton" name="gpsNormal">
         <property name="text">
          <string>GPS闲时间隔</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="apnButton">
         <property name="text">
          <string>设置APN</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QPushButton" name="openGps">
         <property name="text">
          <string>开GPS</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QPushButton" name="uartButton">
         <property name="text">
          <string>关闭串口</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextEdit</class>
   <extends>QWidget</extends>
   <header>textedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>sendButton</tabstop>
  <tabstop>portReadAscii</tabstop>
  <tabstop>portReadHex</tabstop>
  <tabstop>wrapLineBox</tabstop>
  <tabstop>portWriteAscii</tabstop>
  <tabstop>portWriteHex</tabstop>
  <tabstop>resendBox</tabstop>
  <tabstop>resendIntervalBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
